tools/swig/pyfragments.swg

```diff
@@ -52,7 +52,7 @@
     }
 %#endif
     if (!PyArray_IsScalar(obj,Integer)) return SWIG_TypeError;
-    PyArray_Descr * longDescr = PyArray_DescrNewFromType(NPY_LONG);
+    PyArray_Descr * longDescr = PyArray_DescrFromType(NPY_LONG);
```

Comment:
`PyArray_DescrFromType` cannot fail for builtin types (except char)... No idea if it matters, but this is fine.

Reply:
Makes me wonder about the use of `PyArray_DescrNewFromType`. AFAICT, almost all uses are with builtin types.

Reply:
There are about 14 uses for types that aren't mutable. Looks like an opportunity for some cleanup.